### ISSUE TITLE
Community Bounty #24 - "Upload Content"

### ISSUE THREAD

**Issue by bwhm** [Joystream/community-repo#295]
### Description
We are once again opening a "general" bounty for uploading videos to the platform, but this time, the rules will be both stricter and explicit. As with the previous one, only content that falls into the [Public Domain](https://creativecommons.org/share-your-work/public-domain/pdm), or some of the [Creative Commons](https://creativecommons.org/licenses/) licenses (see [Conditions](#conditions)) assuming the requirements are followed, will be allowed.

Note that the bounty text below is mainly targeting the Council, Curators and Bounty Manager(s) that get assigned. It's expected that the "user-facing" blog post, with explicit instructions, will be drafted and approved.

Some ideas that should be considered can be found [here](https://github.com/Joystream/community-repo/blob/master/bounties-reports/bounty_10_revised_rules_draft.md).

### Tags
- `Content`

#### Reward
The reward structure is decided by the Bounty Manager and the Council. The main constraint is the budget, starting at $300 per week, as Jsgenesis will not reimburse larger amounts for each week. The budget does not carry between council terms, so if it is not fully used in one council term, the remaining amount cannot be used in a subsequent term.

#### Format
This will be an "open" bounty, where all users can contribute without the need to be assigned.

A key factor for the BM/Council/Curators is to decide on a workflow for themselves, and a clear set of rules and requirements for the participants. This should help lower the workload for themselves, and ensuring that no videos that doesn't follow the [Conditions](#conditions) slip through.

##### Workflow
The Curators should be most qualified to ensure a video is not in violation of any licenses, copyrights or terms. This is not the same as grading the quality and "value" of a video, and should be separated.
1. Create a rigid reporting format for the participants, e.g.
  - Provide a template table
  - Ensure all the info needed is included, both for verifying the rules are upheld and grading the value of a video
2. At regular intervals (weekly?), one or more Curators, assigned by the Curator Lead, goes through all new entries, and checks each video based on the rules.
3. The Curator creates a new list/table of the approved contributions only, shared in some convenient way, where they add all the videos that are to be graded by the BM. This list should include a link to the individual forum posts from the participants.
  - To make things easier on themselves, if a single video uploaded from a user appears to be in violation, regardless of the infraction, discard all the videos from that user in this batch
4. Once the Curators has completed verifying all videos for that week, which should be within something like ~72h of the deadline the list is shared with the BM.
5. The BM grades all videos, based on pre-defined criterias, creates spending proposal(s) from the Council, and pays out the winners.

Some optional rules should be considered to keep the work manageable:
- Maximum number of entries from a single user per week
- Cap the max payout for a single user per week

#### Success Events
TL;DR for the participants, to be improved!

1. Find interesting videos you want to upload to the platform.
2. Ensure the video license is in line with the [Conditions](#conditions)
  - If you want to modify it, make sure it's not CC BY-ND
3. Create or source the thumbnail you need, and depending on the license type, get the correct [attribution info](https://wiki.creativecommons.org/wiki/License_Versions#Detailed_attribution_comparison_chart).
  - Note that if the source is a YouTube video, with the `License
Creative Commons Attribution license (reuse allowed)` link in the bottom of the description, the version is CC BY 3.0, not 4.0.`
4. Upload the video to the platform, with all the metadata.
5. Repeat 1-4 until you've uploaded all the videos you want for the week.
6. Make a post in the correct forum thread created for the Bounty, with all the information requested, in the exact format required by the BM.

#### Grading
Some examples of grading

Videos are graded on:
- quality of video/audio
- quality of metadata for the video:
  - thumbnail
  - descriptions
  - release year
  - etc.
- quality of metadata for the channel:
  - cover
  - avatar
  - title
- length of content

#### Conditions
- The video, and all metadata, is licensed under:
  - No rights reserved:
    - [Public Domain](https://creativecommons.org/share-your-work/public-domain/pdm)
    - [CC0 - Public Domain Dedication](https://creativecommons.org/share-your-work/public-domain/cc0) (Same terms as the Public Domain)
  - Other Creative Commons licenses:
    - [CC BY - Creative Commons Attribution License](https://creativecommons.org/licenses/by/4.0)
      - Requires attribution
    - [CC BY-SA - Creative Commons Attribution-ShareAlike License](https://creativecommons.org/licenses/by-sa/4.0)
      - Requires attribution, and any derivative work _must_ have the same license
    - [CC BY-ND - Creative Commons Attribution-NoDerivs License](https://creativecommons.org/licenses/by-bd/4.0)
      - Requires attribution, and no derivative work allowed
  - If you want to share any content that is not licensed under any of these, you must clarify this with the Bounty Manager
  - The same licensing rules to all metadata, meaning channel artwork and video thubmnails
- All metadata is set correctly, especially if relevant to the license (eg. attribution is required for all CC BY* licenses)
- Duplicates will not be accepted unless the quality is significantly better
- Videos are placed in channels themed appropriately (eg. a cryptocurrency video shouldn't be in a baking channel)

#### Annihilation
- Anything that violates the platforms ToS and/or any license or copyright, will mean voiding all rewards, not just for that single upload, but for the entire period. This includes, but is not limited to:
  - A video licensed as CC BY, but doesn't have the correct attribution or is published with an incorrect license
  - A video licensed as CC BY-SA, but doesn't have the correct attribution, or has been edited and is then published under a different license on Joystream
  - A video licensed as CC BY-SA, but doesn't have the correct attribution, is published with an incorrect license, or has been edited
  - Further note that just because you find a video licensed as CC BY _somewhere_, it's only valid if it was in fact the owner/creator of the video that published it there. Although this can be difficult to figure out, the burden of proof is on the uploader
- If the BM and the Council approves a single video for rewards, that turns out to be in violation of said terms, the entire rewards for that week will be annihilated.

#### Review Period
After the end of the round (1 week?), the Curators will spend up to 72h to verify the videos are in line with the terms. Then, the BM will have a further 72h to grade them. The individual rewards should then be paid within 24h, for a total of 1 week.

#### Notes
This draft contains a mix of rules and suggestions for the final bounty text. The Council, together with the Curator Lead, needs to agree on a final set of rules both to be applied "internally" and for the participants.

Some rules can only be loosened after seeking approval from Jsgenesis:
- The budget ($300 per week)
- The [Conditions](#conditions)
- The [Annihilation](#annihilation)
- Keeping records of the approved videos, with all the relevant information available
- Keeping records of the rewards each video received, and some notes about the grading

Before the bounty can be opened, the complete set of rules must be presented to, and approved by Jsgenesis.
Steps:
1. Assign a BM
2. Have the BM, alongside a representative of the Curators (either the Lead, or selected by the Lead), draft up the rules
3. Present them for the Council, and collect feedback
4. Once approved by the Council, seek approval from Jsgenesis
5. Once approved by Jsgenesis, open the Bounty

#### Status
Draft. Modifications of _these_ rules may be modified based on Council inputs.
The rules for participants are to be made by the Council/BM, based on this issue.

**Comment by mochet** [comment]
This has been updated to be bounty 24 due to another approved bounty taking the previous number

**Comment by mochet** [comment]
This bounty is no longer relevant to the new system, closing this to avoid confusion
